src/ServiceManagement/Common/Commands.ServiceManagement.Common/packages.config

```diff
@@ -14,5 +14,6 @@
   <package id="Microsoft.Rest.ClientRuntime" version="2.3.5" targetFramework="net45" />
   <package id="Microsoft.Rest.ClientRuntime.Azure.Authentication" version="2.2.9-preview" targetFramework="net45" />
   <package id="Microsoft.WindowsAzure.Management" version="4.1.1" targetFramework="net45" />
+  <package id="Microsoft.WindowsAzure.Management.Compute" version="14.0.0" targetFramework="net45" />
```

Comment:
Remove this.  This project does not need Microsoft.WindowsAzure.Management.Compute